data/campaigns/Descent_Into_Darkness/scenarios/01_Saving_Parthyn.cfg

```diff
@@ -339,6 +339,7 @@
         [message]
             speaker=narrator
             message= _ "Ignorance is the parent of fear."
+            image=wesnoth-icon.png
```

Comment:
It's possible to shut up wmllint while not providing a picture, with
`image=none` (special keyword since 1.14.x)